dirs_cleaner: add page (#21995)
tldr-pages/tldr: pages/osx/dirs_cleaner.md

# dirs_cleaner

> > Recursively deletes the entire contents of each directory argument while the directories themselves are not deleted.

Reviewer: ```suggestion
> Recursively empty directories while the directories themselves are not deleted.
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood
